harmony_changelog.md

```diff
@@ -34,6 +34,7 @@
 
 ### Fixed
 * Brightness calculation for alarms
+* Changed UI flow in pre-wake up 
```

Comment:
Should this entry be here? AFAIU this is a fix for the potential regression, i.e. nothing have changed WRT the previous version.

Reply:
Done!